apps/backend/src/models/proxy_config.py

```
@@ -115,7 +111,7 @@ def validate_names(self, key, value):
         return value.strip().lower()
 
     @validates("file_path")
-    def validate_file_path(self, key, value):
+    def validate_file_path(self, key, value) -> str:
```

Ruff (0.12.2)

Unused method argument: `key`

(ARG002)